samples/languages/java/test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+test_mode=$1
+run_mode=$2
+
+set -e
+
+if [[ "$test_mode" == "clean" ]]; then
+    make clean
+elif [[ "$test_mode" == "init" ]]; then
+    make
+elif [[ "$test_mode" == "run" ]]; then
+
```

Comment:
looks like you left a blank line here accidentally.

Reply:
Fixed.